### DISCUSSION TITLE
Model Deployment Video 299 Gradio Insallation

### DISCUSSION THREAD

**Discussion by SaturoGojo1** [mrdbourke/pytorch-deep-learning#699] [Q&A] [answered]
Hello Everyone, I'm facing some problem regarding importing Gradio. 
![Gradio Error](https://github.com/mrdbourke/pytorch-deep-learning/assets/103381649/0453116c-1a0f-46d3-aa18-1a4422ba29cc)

Did anyone else face this problem, if so any advice will be helpful.
Thanks In Advance.

**Comment by mrdbourke** [comment, marked as answer]
Hi @SaturoGojo1 , 

I just tried to install/import Gradio in a new Colab notebook and it worked fine for me.

Potentially try to reinstall/restart your runtime and see what happens?

```python
!pip install gradio
import gradio as gr
```

Source: https://colab.research.google.com/drive/1f2-ptPGecjPQ0zMNnS6rTVyHmR8i6izv?usp=sharing

<img width="1154" alt="Screenshot 2023-10-24 at 8 31 17 am" src="https://github.com/mrdbourke/pytorch-deep-learning/assets/16750345/cb971edf-46d3-4fa6-8718-c415f348986f">

**Reply by SaturoGojo1** [reply]
@mrdbourke I tried reinstalling and restarting the runtime, and it worked. I don't know what the issue was before, but it works now. Thanks

**Reply by mrdbourke** [reply]
Glad to hear it!
